packages/eslint-plugin/src/rules/jsx-import.js

```diff
@@ -7,9 +8,66 @@ const JSX_ANNOTATION_REGEX = /\*?\s*@jsx\s+([^\s]+)/
 
 export default {
   meta: {
-    fixable: 'code'
+    fixable: 'code',
+    schema: {
+      type: 'array',
+      items: {
+        oneOf: [
+          {
+            type: 'string'
+          },
+          {
+            type: 'object',
+            properties: {
+              runtime: { type: 'string' },
+              importSource: { type: 'string' }
+            },
+            required: ['runtime'],
+            additionalProperties: false
+          }
+        ]
+      },
+      uniqueItems: true,
+      minItems: 0
+    }
   },
   create(context) {
+    const jsxRuntimeMode = context.options.find(
+      option => option && option.runtime === 'automatic'
+    )
+
+    if (jsxRuntimeMode) {
+      return {
+        JSXAttribute(node) {
+          if (node.name.name !== 'css') {
+            return
+          }
+          let hasJsxImportSource = false
+          let sourceCode = context.getSourceCode()
+          let pragma = sourceCode
+            .getAllComments()
+            .find(node => JSX_IMPORT_SOURCE_REGEX.test(node.value))
+          hasJsxImportSource =
+            pragma && pragma.value.match(JSX_IMPORT_SOURCE_REGEX)
```

Comment:
Can you check that the importSource is the one that the user specified?

Reply:
Thanks for code review 🤟  , apply your suggestion later.

Reply:
@mitchellhamilton I apply your suggestion now, code review please, thanks!